src/main/java/co/rsk/federate/btcreleaseclient/cache/PegoutSignedCache.java

```diff
@@ -0,0 +1,26 @@
+package co.rsk.federate.btcreleaseclient.cache;
+
+import co.rsk.crypto.Keccak256;
+
+public interface PegoutSignedCache {
+
+  /**
+   * Checks if the specified RSK transaction hash for pegout creation has already
+   * been signed.
+   * 
+   * @param pegoutCreationRskTxHash The Keccak256 hash of the RSK transaction for
+   *                                pegout creation.
+   * @return {@code true} if the transaction hash has already been signed,
```

Comment:
*if that transaction has already been signed
a bit nit, but more accurate to say that what was signed was a transaction (not a transaction hash)

Reply:
I changed the wording, let me know what you think